index.html

```diff
@@ -6,13 +6,29 @@
   <link href='game.css' rel='stylesheet'/>
 </head>
 <body>
+  <div class='game-area'>
 
-  <canvas id='game' width='600' height='600'>
-    Flappy Bird
-  </canvas>
+    <p class='instructions'>
+      Press 'Space' or click the mouse to flap. </br>
+      Avoid the obstacles!
+    </p>
 
-  <h3>Score: <span class='score'>0</span></h3>
+    <canvas id='game' width='600' height='600'>
+      Flappy Bird
+    </canvas>
 
+    <div class='difficulty-div'>
```

Comment:
I meant in the class name. So just call it `difficulty` or `difficulty-container` if you have to. The `-div` is unnecessary.